web/quickfeed_service.go

```diff
@@ -397,12 +440,16 @@ func (s *QuickFeedService) RebuildSubmissions(_ context.Context, in *connect.Req
 
 // CreateBenchmark adds a new grading benchmark for an assignment.
 func (s *QuickFeedService) CreateBenchmark(_ context.Context, in *connect.Request[qf.GradingBenchmark]) (*connect.Response[qf.GradingBenchmark], error) {
-	bm, err := s.createBenchmark(in.Msg)
-	if err != nil {
+	if _, err := s.db.GetAssignment(&qf.Assignment{
```

Comment:
Can this `GetAssignment` be done first in `CreateBenchmark`?... Or are we using `CreateBenchmark` somewhere else, where we do not need to get the assignment first?